packages/salesforcedx-vscode-core/src/commands/forceLightningAppCreate.ts

```diff
@@ -92,7 +88,7 @@ const fileNameGatherer = new SelectFileName();
 const lightningFilePathExistsChecker = new LightningFilePathExistsChecker();
 
 export async function forceLightningAppCreate(explorerDir?: any) {
-  const outputDirGatherer = new SelectStrictDirPath(explorerDir, 'aura');
+  const outputDirGatherer = new SelectOutputDir('aura', true);
```

Comment:
Because aura bundles require the "aura" parent folder, we let the constructor know this requirement by specifying "true".